libsnowflakeclient/include/snowflake/Simba_CRTFunctionSafe.h

```diff
@@ -1,10 +1,6 @@
-// =================================================================================================
-///  @file Simba_CRTFunctionSafe.h
-///
-///  C Run-Time library functions' definitions for cross platform use.
-///
-///  Copyright (C) 2019 Simba Technologies Incorporated.
-// =================================================================================================
+/*
+ * Copyright (c) 2019-2024 Snowflake Computing, Inc. All rights reserved.
```

Comment:
let's wait for https://github.com/snowflakedb/libsnowflakeclient/pull/667 and include also the changes from this PR

Reply:
We could do that but that would cause delay on the release and there won't be any functionality impact with that.

Reply:
But it means that we are not using exactly libsnowflakeclient 1.0.7 - it should not happen

Reply:
Updated with latest changes in 1.0.7